flytekit/__init__.py

```diff
@@ -280,6 +281,17 @@ def new_context() -> Generator[FlyteContext, None, None]:
     return FlyteContextManager.with_context(FlyteContextManager.current_context().new_builder())
 
 
+def cwd() -> Path:
```

Comment:
Do you thinking this will be confusing for users? The "current working directory" is usually `/root`, but this "working directory" is a random temporary directory.